### PR TITLE
fix(ci): wave @greut

### PR DESCRIPTION
**Pull request by greut**
btw, @iDmple will be on holidays soon, who wants to step in?

### CODE REVIEW

**Review by floriansipp — DISMISSED** [review]
🫡

**Comment by greut** [comment]
@jdaln would you be up for the task? It's about bumping the version number and praying it doesn't break anything. Reading the changelog is advised but not mandatory.

**Comment by jdaln** [comment]
Yes add me. I can't suggest changes without forking in the current permissions so I just comment here.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @iDmple @greut @floriansipp
+* @iDmple @floriansipp
```
Comment (greut) [inline .github/CODEOWNERS:1]:
```suggestion
* @iDmple @floriansipp @jdaln 
```

**Comment by greut** [comment]
ping @iDmple

**Comment by jdaln** [comment]
What is needed @iDmple is to give me write access and then merge this

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-* @iDmple @greut @floriansipp
+* @iDmple @floriansipp @jdaln
```
Comment (greut) [inline .github/CODEOWNERS:1]: then, let's keep it simple for now.

```suggestion
* @iDmple @floriansipp
```

**Comment by iDmple** [comment]
So long @greut, all the best!

**Pull request merged**